.github/workflows/format.yml

```diff
@@ -1,13 +1,16 @@
 name: Format
 
 on:
-  push:
-    branch: 'master'
```

Comment:
Maybe this `branch` was the issue. Looks like it should be `branches`... 🤔

Reply:
lol, should we start using [dhall](https://github.com/vmchale/github-actions-dhall)? 😬